src/components/FileList.jsx

```diff
@@ -21,10 +22,11 @@ const FileList = ({
       <div className={classNames(styles['fil-content-header'], styles['fil-content-status'])}>{ t('table.head_status') }</div>
     </div>
     <div className={styles['fil-content-body']}>
+      <AddFolder />
       {!error && files.map((file, idx) => (
         <File
+          context={context}
           onEdit={onFileEdit}
-          onEditAbort={onFileEditAbort}
```

Comment:
I hesitated to remove it completely: like you said, today it's impossible to edit a filename because of the click/double-click behavior on the row. I opted to not remove everything because the golbal refactoring is still in progress, and I hope to find a functional solution when our PO comes back next week.